test_util/aws.py

```diff
@@ -64,9 +64,12 @@ def create_stack(self, name, template_url, user_parameters, deploy_timeout=60):
 
 
 class CfStack():
-    def __init__(self, stack_name, boto_wrapper):
+    def __init__(self, stack_name, boto_wrapper, ssh_user=None, ssh_home_dir=None):
```

Comment:
I tried to put SSH info as close as possible to the operating system, since that's what determines it. To figure out SSH details you'd have to interrogate the stack object to get its OS, so I figured it might as well answer the question directly.